include/tvm/relay/error.h

```diff
@@ -12,8 +12,12 @@
 namespace tvm {
 namespace relay {
 
+#define RELAY_ERROR(msg) std::stringstream() << msg
+
 struct Error : public dmlc::Error {
-  explicit Error(const std::string &msg) : dmlc::Error(msg) {}
+  Span sp;
+  explicit Error(const std::string &msg) : dmlc::Error(msg), sp() {}
+  Error(const std::stringstream& msg) : dmlc::Error(msg.str()), sp() {}
```

Comment:
explicit. also can leave out sp() as it is the default constructor.

Reply:
I left off the explicit so that the error reporting code can be more elegant.